### PR TITLE
feat: add voice server update

### PR DESCRIPTION
**Pull request by LordOfPolls**
This was missing and i needed it

### CODE REVIEW

**Review by silasary — APPROVED** [review]

File: discord_typings/resources/voice.py [hunk 1]
```diff
@@ -41,3 +41,13 @@ class VoiceRegionData(TypedDict):
     optimal: bool
     deprecated: bool
     custom: bool
+        
+        
+# https://discord.com/developers/docs/topics/gateway#voice-server-update-voice-server-update-event-fields
+
+
+@final
+class VoiceServerData(TypedDict)
```
Comment (silasary) [inline discord_typings/resources/voice.py]: You're missing a colon:

**Review by silasary — CHANGES_REQUESTED** [review]

File: discord_typings/resources/voice.py (hunk 1 above)
Comment (Bluenix2) [reply discord_typings/resources/voice.py]: Yes this is what appears to be causing the lint failure.

Comment (Bluenix2) [inline discord_typings/resources/voice.py]: This class also needs to be added to `__all__`, just add `'VoiceServerData'` to the tuple at the top.

Without doing so, it won't be exposed under the `discord_typings` namespace correctly.

**Comment by LordOfPolls** [comment]
Serves me right for doing this on my phone. I'll fix the mistake shortly

**Comment by Bluenix2** [comment]
When you're fixing that accidental blank space can you also move it to `gayeway.py` with the other events and create a wrapper with the `op`, `d`, `s` and `t` fields?

**Comment by Bluenix2** [comment]
Hello, were you planning on completing this PR? I have opened issue #5 for all events - which I am planning on completing soon - so can I close this?

**Pull request closed without merging**